src/checkpoint.rs

```diff
@@ -139,7 +139,9 @@ mod tests {
         let object_store: Arc<dyn ObjectStore> = Arc::new(InMemory::new());
         let path = Path::from("/tmp/test_kv_store");
         // open and close the db to init the manifest and trigger another write
-        let db = Db::open_with_opts(path.clone(), DbOptions::default(), object_store.clone())
+        let db = Db::builder(path.clone(), object_store.clone())
+            .with_settings(Settings::default())
```

Comment:
Is this needed? Seems like settings uses Settings::default already?

Reply:
Updated.